AdyenActions/UI/View Controllers/DelegatedAuthentication/DelegatedAuthenticationView.swift

```diff
@@ -0,0 +1,325 @@
+        delegate?.secondButtonTapped()
+    }
+    
+    override internal func layoutSubviews() {
+        super.layoutSubviews()
+        updateAxisBasedOnOrientation()
+    }
+    
+    private func updateAxisBasedOnOrientation() {
+        buttonsStackView.axis = UIDevice.current.orientation.isLandscape ? .horizontal : .vertical
+    }
+    
+    internal func animateImageTransitionToSystemImage(named: String) {
+        let workItem = DispatchWorkItem { [weak self] in
+            guard let self else { return }
+            if #available(iOS 17.0, *) {
+                logoImage.addSymbolEffect(.bounce, options: .repeat(2))
+            }
+            
+            if let replacementImage = UIImage(systemName: named,
+                                              withConfiguration: UIImage.SymbolConfiguration(weight: .ultraLight)) {
+                if #available(iOS 17.0, *) {
+                    logoImage.setSymbolImage(replacementImage, contentTransition: .replace)
+                } else {
+                    UIView.transition(with: logoImage,
+                                      duration: 0.5,
+                                      options: .transitionCrossDissolve,
+                                      animations: { self.logoImage.image = replacementImage },
+                                      completion: nil)
+                }
+            }
+        }
+        DispatchQueue.main.asyncAfter(deadline: .now() + 1, execute: workItem)
+    }
+}
+
+// MARK: - view creation helpers
+
+@available(iOS 16.0, *)
+extension UIStackView {
+    internal static func make(arrangedSubviews: [UIView],
+                              axis: NSLayoutConstraint.Axis = .vertical,
+                              distribution: UIStackView.Distribution = .fill,
+                              alignment: UIStackView.Alignment = .fill,
+                              spacing: CGFloat,
+                              view: UIView,
+                              withBackground: Bool = false) -> UIStackView {
+        let stackView = UIStackView(arrangedSubviews: arrangedSubviews)
+        stackView.axis = axis
+        stackView.alignment = alignment
+        stackView.spacing = spacing
+        stackView.distribution = distribution
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        
+        if withBackground {
+            let subView = UIView(frame: view.bounds)
+            subView.backgroundColor = UIColor.secondarySystemBackground
+            subView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+            stackView.insertSubview(subView, at: 0)
+            subView.layer.cornerRadius = 10.0
+            subView.layer.masksToBounds = true
+            subView.clipsToBounds = true
+        }
+        return stackView
+    }
+}
+
+@available(iOS 16.0, *)
+extension UILabel {
+    internal static func make(style: TextStyle, accessibilityPostfix: String, multiline: Bool = false) -> UILabel {
```

Comment:
For all these extensions here:
- Why not init but a static function?
- Why is there a need for iOS 16?
- Let's move them to a separate extension file.

Reply:
I've moved them in separate files. Please have a look.